.github/workflows/solidity-foundry-artifacts.yml

```diff
@@ -129,16 +129,17 @@ jobs:
         with:
           working-directory: contracts
 
-      - name: Copy modified changesets
-        if: ${{ needs.changes.outputs.changeset_changes == 'true' }}
-        run: |
-          mkdir -p contracts/changesets
-          files="${{ needs.changes.outputs.changeset_files }}"
-          IFS=",'
-          for changeset in $files; do
-            echo "::debug:: Copying $changeset"
-            cp $changeset contracts/changesets
-          done
+      # THIS IS BROKEN
+      # - name: Copy modified changesets
+      #   if: ${{ needs.changes.outputs.changeset_changes == 'true' }}
+      #   run: |
+      #     mkdir -p contracts/changesets
+      #     files="${{ needs.changes.outputs.changeset_files }}"
+      #     IFS=",'
+      #     for changeset in $files; do
+      #       echo "::debug:: Copying $changeset"
+      #       cp $changeset contracts/changesets
+      #     done
```

Comment:
how is it broken? can you share the error?

Reply:
The CSV handling does not work, I'll open a ticket for it so we dont lose track of it

Reply:
See https://smartcontract-it.atlassian.net/browse/TT-1506